install_files/ossec-server/var/ossec/rules/local_rules.xml

```diff
@@ -16,3 +19,88 @@
     <description>A cron-apt error was detected</description>
   </rule>
 </group>
+
+<!--
+  Alert on apparmor denied events
+-->
+<group name="apparmor">
+  <rule id="100010" level="0">
+    <decoded_as>iptables</decoded_as>
+  </rule>
+
+  <rule id="100011" level="0">
+    <if_sid>100010</if_sid>
+    <status>STATUS</status>
+    <description>Apparmor status event</description>
+  </rule>
+
+  <rule id="100012" level="7">
+    <if_sid>100010</if_sid>
+    <status>DENIED</status>
+    <description>Apparmor denied event</description>
+  </rule>
+
+  <rule id="100013" level="0">
+    <if_sid>100010</if_sid>
+    <status>ALLOWED</status>
+    <description>Apparmor allowed event</description>
+  </rule>
+</group>
+
+<!--
+  Grsec related rules. Info and expected events should be whitelisted.
+  All other grsec events should be alerted on.
+-->
+<group name="grsec">
+  <rule id="100101" level="7">
+    <decoded_as>grsec</decoded_as>
+    <description>grsec error was detected</description>
+  </rule>
+
+  <rule id="100102" level="0">
+    <if_sid>100101</if_sid>
+    <match>^time set by</match>
+    <options>no_email_alert</options>
+  </rule>
+</group>
+
+<!--
+  Do not alert on stagging VM dhcp client errors. These events should not occur
+  in production environments
+-->
+<group name="stagging vm errors">
+  <rule id="110030" level="0">
+    <decoded_as>dhclient</decoded_as>
+    <match>Failed to send 300 byte long packet over fallback interface.</match>
+    <options>no_email_alert</options>
+  </rule>
+</group>
+
+<!--
+  The python gnupg library that securedrop uses includes an obsolete option
+  WARNING:gnupg no-use-agent is an obsolete option - it has no effect
+  Do not send an alert for this event.
+-->
+<group name="do not alert">
+  <rule id="199997" level="0">
+    <decoded_as>gpg warning</decoded_as>
+    <description>due to upstream lib</description>
+    <options>no_email_alert</options>
```

Comment:
Should add a <match> for the specific `no-use-agent is an obsolete option` event so that other possible gpg warning events don't get suppressed.